File: Scrapers/lasseshifi.py
from Scrapers.common import ShopifyCollectionScraper


class LassesHiFiScraper(ShopifyCollectionScraper):
    def __init__(self):
        super().__init__(
            base_url="https://lasseshifi.se",
            name="Lasses HiFi",
            collection_path="/collections/erbjudande",
        )

File: ts-scrapers/src/sites/lasseshifi.ts
import { ShopifyCollectionScraper } from '../bases/shopify.js';

export class LassesHiFiScraper extends ShopifyCollectionScraper {
  constructor() {
    super({
      name: 'Lasses HiFi',
      baseUrl: 'https://lasseshifi.se',
      collectionPath: '/collections/erbjudande',
      slug: 'lasses-hifi',
    });
  }
}
